### PR TITLE
Add guideline for choosing an iterative solver

### PR DESCRIPTION
**Pull request by stefanv**
Addresses #129

### CODE REVIEW

File: markdown/ch6.markdown [hunk 1]
```diff
@@ -655,6 +655,39 @@ plot_connectome(x, y, C, labels=neuron_ids, types=neuron_types,
 
 <!-- exercise end -->
 
+## SIDEBOX: Solvers
+
+SciPy has several sparse iterative solvers available, and it is not
+always obvious which to use.  Unfortunately, that question also has no
+easy answer: different algorithms have different strengths in terms
+of, e.g., speed of convergence, stability, accuracy, and memory use.
```
Comment (jni) [inline markdown/ch6.markdown]: remove e.g.

File: markdown/ch6.markdown [hunk 2]
```diff
@@ -655,6 +655,39 @@ plot_connectome(x, y, C, labels=neuron_ids, types=neuron_types,
 
 <!-- exercise end -->
 
+## SIDEBOX: Solvers
```
Comment (jni) [inline markdown/ch6.markdown]: We should probably unify how we deal with these... Currently it's been block-quoted and had a `{.callout}` class annotation...

**Comment by stefanv** [comment]
Taking out the e.g. changes the meaning of the sentence.  I want to
say: there are many factors to take into consideration, these are a
few of them.

**Comment by jni** [comment]
@stefanv sure, but as it is it really breaks the cadence. Perhaps "in terms of a, b, and c, among others"?

**Comment by stefanv** [comment]
I'm going to bed now, but feel free to amend according to preference and merge!

**Comment by jni** [comment]
@stefanv you ignored my comment about formatting so I'm gonna wait till you wake up. =P

**Comment by jni** [comment]
@stefanv how does the current formatting of the SIDEBOX fit in with the ATLAS build? Does it need to be updated?

**Comment by stefanv** [comment]
Fixed.

**Pull request merged**